python/python/ert_gui/plottery/plot_data_gatherer.py

```diff
@@ -72,11 +72,12 @@ def gatherSummaryData(ert, case, key):
         if not data.empty:
             data = data.reset_index()
 
-            if True in data.duplicated():
-              print("** Warning: The simulation data contains duplicate "
-                    "timestamps. A possible explanation is that your "
-                    "simulation timestep is less than a second.")
-              data = data.drop_duplicates()
+            if True in any(data.dupliated()):
```

Comment:
This should be 
`if any(data.dupliated()):`

Reply:
Thank you.